src/actions/google/drive/sheets/test_google_sheets.ts

```diff
@@ -538,6 +538,15 @@ describe(`${action.constructor.name} unit tests`, () => {
       })
     })
 
+    describe("delay", () => {
+      it("will block on await delay", async () => {
```

Comment:
Can you provide more detail for why this is necessary?

Reply:
the previous implementation, although it looked correct, was not actually delaying retries. I am seeing another area which I missed so I will upload another change to this as well